src/app/public/modules/flyout/flyout.component.spec.ts

```diff
@@ -869,4 +869,110 @@ describe('Flyout component', () => {
       expect(flyout.iteratorPreviousButtonDisabled).toEqual(false);
     }));
   });
+
+  describe('flex classes', () => {
+    it('should add the xs class when appropriate', fakeAsync(() => {
+      openFlyout({ maxWidth: 10000, minWidth: 50 });
+      fixture.detectChanges();
+      tick();
+      const flyoutElement = getFlyoutElement();
+      const handleElement = getFlyoutHandleElement();
+
+      expect(flyoutElement.style.width).toBe('500px');
```

Comment:
Haven't we already established flyout initial width in prior tests? Why repeat it in each test?

Reply:
Yup. They were just there from copy/paste :)

Removed :)